python-checks/src/main/java/org/sonar/python/checks/hotspots/ClearTextProtocolsCheck.java

```diff
@@ -89,28 +85,26 @@ public void initialize(Context context) {
 
     context.registerSyntaxNodeConsumer(Tree.Kind.QUALIFIED_EXPR, ClearTextProtocolsCheck::checkServerCallFromSuper);
 
-    context.registerSyntaxNodeConsumer(Tree.Kind.FUNCDEF, ClearTextProtocolsCheck::checkServerBindOverrides);
+    context.registerSyntaxNodeConsumer(Tree.Kind.CALL_EXPR, ClearTextProtocolsCheck::checkServerBindCalls);
+
     new ClearTextProtocolsCheckPart().initialize(context);
   }
 
   private static void checkServerCallFromSuper(SubscriptionContext ctx) {
     QualifiedExpression qualifiedExpression = (QualifiedExpression) ctx.syntaxNode();
     Optional.of(qualifiedExpression)
-      .filter(qe -> isName("serve_forever", qe.name()) && isCallToSensitiveSuperClass(qe))
+      .filter(qe -> SENSITIVE_HTTP_SERVER_METHOD_NAMES.contains(qe.name().name()) && isCallToSensitiveSuperClass(qe))
```

Comment:
Maybe here it would be worth splitting this into two separate filters:
```java
.filter(qe -> SENSITIVE_HTTP_SERVER_METHOD_NAMES.contains(qe.name().name()))
  .filter(ClearTextProtocolsCheck::isCallToSensitiveSuperClass)
```